assets/css/sidebar.css

```diff
@@ -72,6 +72,8 @@
     top: 0;
     height: 100%;
     padding: 20px;
+    padding-right:5px !important;
```

Comment:
Is the `!important` for padding right and top necessary? I think it would work without them. Please try and if it doesn't then you can leave them